docker/pravega-schema-registry/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+FROM openjdk:8u212-jre-alpine3.9
+
+RUN apk add --update \
+    rpcbind \
+    nfs-utils \
+    gcompat \
+    python \
```

Comment:
As i understand packages like nfs-utils are not required. Could we review if these packages are required.?

Reply:
removed nfs-util.i dont think we need any of the packages by default as such. python jq net tools etc are good to have if someone wants to ssh into the pod and investigate

Reply:
python is being used by `wait_for` file